### PR TITLE
reporter: Extract CSS of static HTML report to a file

### PR DESCRIPTION
**Pull request by mnonnenmacher**
This improves overview in the Kotlin file and enables syntax
highlighting for the CSS file when opening in an IDE.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1416)

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt [hunk 1]
```diff
@@ -39,262 +39,7 @@ import kotlinx.html.*
...
-              border-right: 1px solid rgba(34,36,38,.15);
-            }
-
-            .ort-report-table tr:last-child td:first-child {
-              border-bottom-right-radius: .28rem;
-            }
-        }
-
-        """.trimIndent()
+    private val css = javaClass.getResource("/static-html.css").readText()
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: Nit: Maybe name the file `static-html-reporter.css`?

Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: Done.

File: reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt [hunk 2]
```diff
@@ -324,6 +69,7 @@ class StaticHtmlReporter : Reporter() {
                 meta(name = "viewport", content = "width=device-width, initial-scale=1.0")
                 title("Scan Report")
                 style {
+                    +"\n"
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: This seems to cause a compiler warning.

Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: Fixed.

**Pull request merged**